_stbt/imgutils.py

```diff
@@ -157,6 +159,111 @@ def _frame_repr(frame):
         return "<%dx%d>" % (frame.shape[1], frame.shape[0])
 
 
+class Color:
+    """A BGR color, optionally with an alpha (transparency) value.
+
+    A Color can be created from an HTML-style hex string:
+
+    >>> Color('#f77f00')
+    Color('#f77f00')
+
+    Or from Blue, Green, Red values in the range 0-255:
+
+    >>> Color(0, 127, 247)
+    Color('#f77f00')
+
+    Note: When you specify the colors in this way, the BGR order is the
+    opposite of the HTML-style RGB order. This is for backwards compatibility
+    with the way OpenCV stores colors.
+    """
+    @overload
+    def __init__(self, hexstring: str) -> None:
+        ...
+    @overload
+    def __init__(self, blue: int, green: int, red: int) -> None:
+        ...
+    @overload
+    def __init__(self, bgr_tuple: Tuple[int, int, int]) -> None:
+        ...
+    @overload
+    def __init__(self, bgra_tuple: Tuple[int, int, int, int]) -> None:
+        ...
+    def __init__(self, *args):
```

Comment:
> Does this complete nicely in VSCode?

See comment below: https://github.com/stb-tester/stb-tester/pull/745#issuecomment-1141806927

> We'd need to keep the conversion from `'#222222'` and `(b, g, r)` because we'll be applying these automatically. Probably not worth changing.

Exactly. The whole point is to have a single place to convert from any accepted input to our internal representation.